go/vt/vttls/vttls.go

```diff
@@ -94,17 +94,23 @@ func ClientConfig(cert, key, ca, name string) (*tls.Config, error) {
 
 // ServerConfig returns the TLS config to use for a server to
 // accept client connections.
-func ServerConfig(cert, key, ca string) (*tls.Config, error) {
+func ServerConfig(cert, key, ca string, combineCerts bool) (*tls.Config, error) {
 	config := newTLSConfig()
 
-	certificates, err := loadTLSCertificate(cert, key)
-
-	if err != nil {
-		return nil, err
+	if ! combineCerts {
```

Comment:
Maybe having the `truthy` case first might be easier to understand for the reader. 

```
if combineCerts {
  bla
} 
else {
 bla
}

Reply:
done, fixe.